package.json

```diff
@@ -38,11 +38,11 @@
     "approve:css": "yarn workspace css-workshop approve",
     "approve:react": "yarn workspace react-workshop approve",
     "clean": "turbo run clean && rimraf node_modules",
-    "release": "yarn build --filter={./packages/*}... && changeset publish"
+    "changeset": "npx -y @changesets/cli",
+    "release": "yarn build --filter={./packages/*}... && yarn changeset publish"
```

Comment:
non-blocking: Is there any reason for using both `yarn` and `npx`? Would it be better to just use one?

Reply:
`yarn changeset` is a shorthand for what was previously `changeset`.

These commands worked before and continue to work now:
```
yarn changeset
yarn changeset version
yarn changeset publish
```

The `npx -y` is an implementation detail of the `changeset` script (L43).